### ISSUE TITLE
Accessibility: https://ipython.org/usersurvey2011.html

### ISSUE THREAD

**Issue by Ibom99** [jupyterhub/outreachy#124]
### What page is this for?

https://ipython.org/usersurvey2011.html

### WAVE accessibility report

![Screenshot (218)](https://user-images.githubusercontent.com/104465078/197482063-0776eda8-40c9-4620-ac67-ab5beedc67e7.png)
![Screenshot (219)](https://user-images.githubusercontent.com/104465078/197482084-a2937582-f000-4d76-84df-d532e5c0d0f0.png)
https://github.com/jupyterhub/outreachy/issues/38#issuecomment-1273856392

### The HTML element to be changed

```html
1.) Errors(1)- Language missing or invalid: 
<html xmlns="http://www.w3.org/1999/xhtml" style="margin-left: 380px;">

2.) Contrast Errors(29): 
The first 18 errors have a lighter orange styling -
<a href="https://jupyter.readthedocs.io/en/latest/install.html" target="_blank" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">
Install
</a>
<a href="documentation.html" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">
Documentation
</a>
<a href="project.html" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">
Project
</a>
<a href="https://jupyter.org/" target="_blank" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">
Jupyter
</a>
<a href="news.html" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">
News
</a>
<a href="citing.html" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">
Cite
</a>
<a href="donate.html" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">
Donate
</a>
<a href="books.html" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">
Books
</a>
<a class="reference external" href="http://stackoverflow.com/questions/tagged/ipython" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">
Stack Overflow
</a>
<a class="reference external" href="https://mail.python.org/mailman/listinfo/ipython-dev" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">
Mailing list
</a>
<a class="reference external" href="https://github.com/ipython/ipython/issues" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">
File a bug
</a>
<a class="reference external" href="http://www.reddit.com/r/IPython" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">
Reddit
</a>
<a class="reference external" href="https://mail.python.org/mailman/listinfo/ipython-dev" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">
Mailing list
</a>
<a class="reference external" href="https://gitter.im/ipython/ipython" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">
Development Chat Room
</a>
<a class="reference external" href="https://github.com/ipython/ipython/wiki/Dev:-Index" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">
Development information
</a>
<a class="reference external" href="http://travis-ci.org/#!/ipython/ipython" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">
Travis CI
</a>
<a class="reference external" href="https://github.com/ipython/ipython/wiki" style="color: rgb(206, 92, 0); background-color: rgb(255, 255, 255);">
Wiki
</a>
<a href="donate.html" style="color: rgb(206, 92, 0); background-color: rgb(225, 232, 236);">
Find out more...
</a>
The next 7 icon colors are a very light gray.
<a class="headerlink" href="#ipython-user-survey-2011" title="Permalink to this headline" style="color: rgb(221, 221, 221); background-color: rgb(255, 255, 255);">
¶
</a>
<a class="headerlink" href="#countries" title="Permalink to this headline">
¶
</a>
<a class="headerlink" href="#platforms" title="Permalink to this headline">
¶
</a>
<a class="headerlink" href="#usage" title="Permalink to this headline">
¶
</a>
<a class="headerlink" href="#sector" title="Permalink to this headline">
¶
</a>
<a class="headerlink" href="#requests" title="Permalink to this headline">
¶
</a>
<a class="headerlink" href="#design-of-the-survey" title="Permalink to this headline">
¶
</a>
The copyright text in the footer is a light grey.
div class="footer" role="contentinfo">
© Copyright the IPython development team. Created using
Very low contrast
<a href="http://sphinx-doc.org/">
Sphinx
</a>

3.) Alerts: Suspicious alternative text(5)-
<img class="logo" src="_static/IPy_header.png" alt="Logo">
<img alt="_images/countries.png" src="_images/countries.png">
<img alt="_images/platforms.png" src="_images/platforms.png">
<img alt="_images/partsused.png" src="_images/partsused.png">
<img alt="_images/sector.png" src="_images/sector.png">

4.) Alerts: Image with title -
<img src="_static/nbviewer-thm-7.png" title="IPython Notebook Viewer">
<img title="Packt Publishing donates a portion of the proceeds from this book to support IPython's development." width="160" height="200" id="sidebar-book-cover" style="margin-bottom: 10px;" src="_static/jupyter-for-ds.png">

5.) Alerts: Suspicious link text(1)-
<a class="reference external" href="https://docs.google.com/spreadsheet/ccc?key=0AqIElKUDQl8tdDMzREtNSk9pX282N3lxaXhiTmxyN0E&authkey=CNOmu-QC&hl=en_GB&authkey=CNOmu-QC">
here
</a>
```

### Your proposed HTML (or CSS), after change

```html
1.) Errors(1)- Language missing or invalid: I will add a `lang` attribute to the `<html>` tag and set the value to `"en"`
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" style="margin-left: 380px;">

2.) Contrast Errors(29): I will change the color inline style to maroon so it increases contrast with the background and leave the background color as it is.
<a href="https://jupyter.readthedocs.io/en/latest/install.html" target="_blank" style="color: rgb(128,0,0); background-color: rgb(255, 255, 255);">
Install
</a>
<a href="documentation.html" style="color: rgb(128,0,0); background-color: rgb(255, 255, 255);">
Documentation
</a>
<a href="project.html" style="color: rgb(128,0,0); background-color: rgb(255, 255, 255);">
Project
</a>
<a href="https://jupyter.org/" target="_blank" style="color: rgb(128,0,0); background-color: rgb(255, 255, 255);">
Jupyter
</a>
<a href="news.html" style="color: rgb(128,0,0); background-color: rgb(255, 255, 255);">
News
</a>
<a href="citing.html" style="color: rgb(128,0,0); background-color: rgb(255, 255, 255);">
Cite
</a>
<a href="donate.html" style="color: rgb(128,0,0); background-color: rgb(255, 255, 255);">
Donate
</a>
<a href="books.html" style="color: rgb(128,0,0); background-color: rgb(255, 255, 255);">
Books
</a>
<a class="reference external" href="http://stackoverflow.com/questions/tagged/ipython" style="color: rgb(128,0,0); background-color: rgb(255, 255, 255);">
Stack Overflow
</a>
<a class="reference external" href="https://mail.python.org/mailman/listinfo/ipython-dev" style="color: rgb(128,0,0); background-color: rgb(255, 255, 255);">
Mailing list
</a>
<a class="reference external" href="https://github.com/ipython/ipython/issues" style="color: rgb(128,0,0); background-color: rgb(255, 255, 255);">
File a bug
</a>
<a class="reference external" href="http://www.reddit.com/r/IPython" style="color: rgb(128,0,0); background-color: rgb(255, 255, 255);">
Reddit
</a>
<a class="reference external" href="https://mail.python.org/mailman/listinfo/ipython-dev" style="color: rgb(128,0,0); background-color: rgb(255, 255, 255);">
Mailing list
</a>
<a class="reference external" href="https://gitter.im/ipython/ipython" style="color: rgb(128,0,0); background-color: rgb(255, 255, 255);">
Development Chat Room
</a>
<a class="reference external" href="https://github.com/ipython/ipython/wiki/Dev:-Index" style="color: rgb(128,0,0); background-color: rgb(255, 255, 255);">
Development information
</a>
<a class="reference external" href="http://travis-ci.org/#!/ipython/ipython" style="color: rgb(128,0,0); background-color: rgb(255, 255, 255);">
Travis CI
</a>
<a class="reference external" href="https://github.com/ipython/ipython/wiki" style="color: rgb(128,0,0); background-color: rgb(255, 255, 255);">
Wiki
</a>
<a href="donate.html" style="color: rgb(128,0,0); background-color: rgb(225, 232, 236);">
Find out more...
</a>
The next 7 icons would be a deeper grey I'll use CSS to select their class.
(CSS)
.headerlink{
color: rgb(104, 104, 104);
}
for the footer: a deeper grey as well.
(CSS)
.footer{
color: rgb(104, 104, 104);
}

3.) Alerts: Suspicious alternative text(5)-
<img class="logo" src="_static/IPy_header.png" alt="IPython Logo">
<img alt="Pie Chart of Respondent Countries" src="_images/countries.png">
<img alt="Bar Graph of the platforms IPython users use." src="_images/platforms.png">
<img alt="Bar Graph of respondents use of the interactive shell." src="_images/partsused.png">
<img alt="Bar Graph of respondents and their various sectors." src="_images/sector.png">

4.) Alerts: Image with title - I would add an `alt` attribute to the `img` tags.
<img src="_static/nbviewer-thm-7.png" title="IPython Notebook Viewer" alt="IPython Notebook Viewer">
<img alt="Jupyter: IPython book publications" title="Packt Publishing donates a portion of the proceeds from this book to support IPython's development." width="160" height="200" id="sidebar-book-cover" style="margin-bottom: 10px;" src="_static/jupyter-for-ds.png">

5.) Alerts: Suspicious link text(1)- added a more descriptive text to represent the destination of the link.
<a class="reference external" href="https://docs.google.com/spreadsheet/ccc?key=0AqIElKUDQl8tdDMzREtNSk9pX282N3lxaXhiTmxyN0E&authkey=CNOmu-QC&hl=en_GB&authkey=CNOmu-QC">
Spreadsheet of the 240 responses
</a>
```

### More info

1.) I would suggest using an external CSS file to the style the color of the texts, to.
2.) Add a `footer` tag for a more semantic HTML codebase.

**Comment by minrk** [comment]
Thanks for the suggestions!

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
